### ISSUE TITLE
Parallel/Dual Dialog in Atom Plug-in Doesn't Appear to Render

### ISSUE THREAD

**Issue by jonathanmschwabe** [superlou/fountain-atom#88]
Using the following example text directly from the fountain page, in both PDF generation and PDF preview, the carot char is basically ignored.

```
The men look at each other.

STEEL
Screw retirement.

BRICK ^
Screw retirement.
```

I tried this several months back and it seemed to work fine.  Unfortunately, I do not recall which version of the plug-in it worked in.

**Comment by superlou** [comment]
This should work in current version by going into settings, spacing, and checking the "Dual dialogue" option. I think we should consider enabling this option by default, since it would be the expected behavior of someone approaching Fountain syntax. Thoughts?

**Comment by jonathanmschwabe** [comment]
Confirmed.  It works per your suggestion.

I would agree.  It is "standard" syntax per the fountion page, so default "on" would be my recommendation.

**Comment by superlou** [comment]
@jrbostic, any thoughts on this, since I think you set up the defaults?

**Comment by jrbostic** [comment]
@superlou Definitely!  I don't see any issue with updating the project defaults other than how it behaves with local defaults that users have already set up.  I'd want to make sure that we don't wipe their changes on version updates.  I seem to remember this coming up in another thread... 

We just need to ensure that there's a local config on the users machine and a package config to fallback on.  Maybe?

**Comment by jonathanmschwabe** [comment]
I would be fine closing this issue if the default configuration is set to "On" going forward.  I'll leave it up to the community, as it works fine once I change the default locally.
